hw_3_zd_3.js

```diff
@@ -0,0 +1,30 @@
+function deleteDublicate(arr) {
+    arr.forEach(function (item, i) {
+        console.log("indexArr: " + i + ": " + "value: " + item.id);
```

Comment:
you have 3 cycles inside the function. This is not optimized.

Reply:
Done